packages/terra-form-checkbox/src/Checkbox.jsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Checkbox.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+    * The checked property of the Input element. Use this to generate a controlled Checkbox Element.
+    */
+  checked: PropTypes.bool,
+  /**
+    * The checked property of the Input element. Use this to generate an uncontrolled Checkbox Element.
+    */
+  defaultChecked: PropTypes.bool,
+  /**
+    * The id of the input field.
+    */
+  id: PropTypes.string,
+  /**
+    * Additional attributes for the input object.
+    */
+  inputAttrs: PropTypes.object,
+  /**
+   * Whether the checkbox element is disabled or not.
+   */
+  isDisabled: PropTypes.bool,
+  /**
+    * Whether the checkbox element is inline or not.
+    */
+  isInline: PropTypes.bool,
+  /**
+    * Whether the label is hidden or not.
+    */
+  isLabelHidden: PropTypes.bool,
```

Comment:
I'm curious, what is the desire for this prop? When would one desire a hidden label?

Reply:
Checkout #973.